pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -850,6 +850,10 @@ self: super: {
     dontBuild = true;
   };
 
+  nvim-navbuddy = super.nvim-navbuddy.overrideAttrs {
+    dependencies = with self; [ nvim-lspconfig nvim-navic  ];
```

looks like we also need nui.nvim

```suggestion
    dependencies = with self; [ nui-nvim nvim-lspconfig nvim-navic ];
```

---

I thought I added this I'm, how strange. Thanks.